### DISCUSSION TITLE
Comments plugin

### DISCUSSION THREAD

**Discussion by brachypelma** [payloadcms/payload#2489] [Show and tell]
Hi Payload enthusiasts,

I wanted to share [a Payload plugin I authored](https://github.com/brachypelma/payload-plugin-comments) that generates a Comments collection with some customizable API endpoints for new comment submission and checking for approved comments. The idea is to make it easy to add comments/reviews functionality to a blog or other site where feedback/discussion of content could be useful.

It is fairly basic as it currently stands, but it seems to be working as intended at the moment.

This is my first time writing a Payload plugin, so I would appreciate any feedback, including issues with the code, bug reports, user experience, or features to add.

**Comment by TechSynthesis** [comment]
Hey @brachypelma, great work. Do you have any frontend implementation of this that I can reference?

**Reply by brachypelma** [reply]
Thank you, @TechSynthesis. The only place I know of where it is up and running is my blog, which is what I developed it for: https://www.confoundedtechnology.com/

The comment form appears at the bottom of a post, e.g.:

https://www.confoundedtechnology.com/post/configuring-dovecot-to-allow-gmail-pop3-access-to-an-email-account/
